src/app/pages/loading-waiting-states/progress-bar-indeterminate/progress-bar-indeterminate.component.html

```diff
@@ -0,0 +1,14 @@
+<mat-toolbar data-cy="pxb-toolbar" flexLayout="row" class="app-bar mat-elevation-z2" [color]="checked ? 'primary' : ''">
```

Comment:
mat-elevation-z4 is the usual elevation we use for app bar

Reply:
Sure let me update.